excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ParseContext.java

```diff
@@ -153,6 +159,29 @@ public ParseContext registerStateForCaseType(String caseTypeId, StateEntity stat
         return this;
     }
 
+    public CategoryEntity getCategory(String caseTypeId, String categoryId) {
+        final Map<String, CategoryEntity> categories = categoryByCaseTypes.get(caseTypeId);
```

Comment:
Is it possible for `categoryByCaseTypes.get(caseTypeId)` can return `emptyMap()` rather than `null` to avoid the following null check?

Reply:
Unfortunately not, it would be nice if it was code under our control, but `categoryByCaseTypes` is an instance of `java.util.Map`, and the get method Javadocs specify

`@return the value to which the specified key is mapped, or {@code null} if this map contains no mapping for the key`